src/main/java/core/basesyntax/service/ReportWriter.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface ReportWriter {
+    boolean writeReport(String report, String filePath);
+}
```

Comment:
краще не прив'язуватись до Report, в майбутньому ми можемо будь-що записувати за допомогоб цього класу у файл
```suggestion
public interface FileWriter {
    boolean write(String content/data/text..., String filePath);
}
```